assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -16,12 +16,16 @@
 /**
  * External dependencies
  */
-import { __ } from '@web-stories-wp/i18n';
+import PropTypes from 'prop-types';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
```

Comment:
Use `import { __ } from '@web-stories-wp/i18n';` here!

Reply:
It's a new addition, so no worries if you haven't seen it :)

Reply:
I'll add a linter rule for this.